packages/lesswrong/components/editor/PostCollaborationEditor.tsx

```diff
@@ -74,20 +74,20 @@ const PostCollaborationEditor = ({ classes }: {
     return <Loading/>
   }
   
-  // If you're the primary author, redirect to the main editor (rather than the
+  // If you're the primary author, or an admin, redirect to the main editor (rather than the
```

Comment:
I think this should redirect _anyone_ with edit permissions.

@jpaddison3 Do you object?

Reply:
Did you mean with*out* edit permissions?

It can't hard-redirect admins or there'd be no way for an admin to edit the post, right? I must be missing something.